helm/dex-app/Chart.yaml

```diff
@@ -26,4 +26,6 @@ maintainers:
     email: nbadger@mintel.com
 annotations:
   application.giantswarm.io/team: team-rainbow
+  application.giantswarm.io/readme: https://raw.githubusercontent.com/giantswarm/dex-app/v[[ .Version ]]/README.md
+  application.giantswarm.io/values-schema: https://raw.githubusercontent.com/giantswarm/dex-app/v[[ .Version ]]/helm/dex-app/values.schema.json
```

Comment:
Just double checking here: are you sure that `[[ .Version ]]` will not include the `v` prefix?

Reply:
I think so - I enabled "replace-chart-version-with-git" for `app-build-suite` and this was part of the output from "package and build chart - Execute App Build Suite" step in CircleCI:
```
Replacing 'version' with git version '1.21.1-cda2bae9375ea335f88b30b5b19ebb89cde525fe' in Chart.yaml.
```
I think that looks correct?